jupyter-docker/Dockerfile

```diff
@@ -121,16 +112,17 @@ RUN export CLOUD_SDK_REPO="cloud-sdk-$(lsb_release -c -s)" \
  && pip install matplotlib-venn \
```

Comment:
I thought pip caches might be something to clean up, but it's not caching anything anyway -- logging this during docker build:
```
The directory '/home/jupyter-user/.cache/pip/http' or its parent directory is not owned by the current user and the cache has been disabled. Please check the permissions and owner of that directory. If executing pip with sudo, you may want sudo's -H flag.
```